api/persistence/models/file.py

```diff
@@ -64,6 +66,8 @@ def FromDBRecord(cls, record:T.NamedTuple, idm:idm.base.IdentityManager) -> File
                    path   = T.Path(record.path),
                    key    = T.Path(record.key) if record.key is not None else None,
                    mtime  = time.to_utc(record.mtime),
+                   atime  = time.to_utc(record.mtime),
+                   ctime  = time.to_utc(record.mtime),
```

Comment:
I think it'd be good to add a comment here, to just explain that the DB only holds the `mtime`, because otherwise someone may look at this later and be a little confused

Reply:
Comment added